service/remotecontrol/RemoteControlEvents.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Events fired from the remote control module trough the EventEmitter.
+ */
+
+/**
+ * Notifies about remote control active session status changes.
+ */
+export const ACTIVE_CHANGED = 'active-changed';
```

Comment:
This is the old architecture. Couple of months ago every constant in meet was defined in service. I created this file there because we already had the Constants.js there.